tools/ARIAtools/product.py

```diff
@@ -662,7 +821,7 @@ def __continuous_time__(self):
             # and spatially overlaps with reference scene
             if (scene_area.intersection(new_scene_area).area > 0. and
                 abs(new_scene_t - scene_t) <= ONE_DAY and
-                abs(new_scene_t_ref - scene_t_ref) <= ONE_DAY):
+                    abs(new_scene_t_ref - scene_t_ref) <= ONE_DAY):
```

Comment:
last line not aligned correctly

Reply:
Apologies, the autopep8 tool I use keeps overriding this. Will fix it.